### PR TITLE
[device/celestica]: Fix sfp index on dx010 platform api

### PR DESCRIPTION
**Pull request by mudsut4ke**
**- What I did**
- Fixed port indexing issue in DX010 sfp api

**- How I did it**
- Used overriding method to fix the port index issue.

**- How to verify it**
- Check sfpshow presence and verify port status is getting updated or not
- Check sfpshow eeprom and verify port eeprom is getting updated or not
- Test log : [dx0101_sfpshow_logs.txt](https://github.com/Azure/sonic-buildimage/files/4414590/dx0101_sfpshow_logs.txt)

### CODE REVIEW

**Comment by mudsut4ke** [comment]
retest default please

**Comment by mudsut4ke** [comment]
retest this please

**Comment by mudsut4ke** [comment]
retest this please

**Comment by mudsut4ke** [comment]
Note: This also at label to merge this PR to 201911 branch

File: device/celestica/x86_64-cel_seastone-r0/sonic_platform/chassis.py [hunk 1]
```diff
@@ -159,3 +158,24 @@ def get_watchdog(self):
...
+        """
+        sfp = None
+
+        try:
+            # The index will start from 1
+            sfp = self._sfp_list[index-1]
+        except IndexError:
+            sys.stderr.write("SFP index {} out of range (1-{})\n".format(
+                             index, len(self._sfp_list)))
+        return sfp
```
Comment (jleveque) [inline device/celestica/x86_64-cel_seastone-r0/sonic_platform/chassis.py:181]: Add newline at EOF

Comment (mudsut4ke) [reply device/celestica/x86_64-cel_seastone-r0/sonic_platform/chassis.py]: done https://github.com/Azure/sonic-buildimage/pull/4346/commits/41801bd83235f712e428746ab6479704c5ed2286

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by lguohan** [comment]
retest vsimage please

**Pull request merged**